Framework/CurveFitting/inc/MantidCurveFitting/ComplexVector.h

```diff
@@ -0,0 +1,153 @@
+#ifndef MANTID_CURVEFITTING_COMPLEXVECTOR_H_
+#define MANTID_CURVEFITTING_COMPLEXVECTOR_H_
+
+#include "MantidCurveFitting/DllConfig.h"
+#include <gsl/gsl_vector.h>
+
+#include <ostream>
+#include <vector>
+#include <complex>
+
+namespace Mantid {
+namespace CurveFitting {
+
+typedef std::complex<double> ComplexType;
```

Comment:
It's a complex type used in `Mantid::CurveFitting`. I think the prefix should only be used if the real number type isn't `double`.